Sources/Nimble/Utils/Async.swift

```diff
@@ -351,4 +358,12 @@ internal func pollBlock(
         }.timeout(timeoutInterval, forcefullyAbortTimeout: timeoutInterval / 2.0).wait(fnName, file: file, line: line)
 
         return result
+    #else
+        do {
+            let pass = try expression()
+            return pass ? .Completed(true) : .TimedOut
+        } catch let error {
+            return .ErrorThrown(error)
+        }
```

Comment:
I'm a bit mixed about even having this around to support `toEventually` when it doesn't really work as expected.